src/test/java/com/endava/internship/mocking/repository/InMemUserRepositoryTest.java

```diff
@@ -0,0 +1,37 @@
+package com.endava.internship.mocking.repository;
+
+import com.endava.internship.mocking.model.Status;
+import com.endava.internship.mocking.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+
+class InMemUserRepositoryTest {
+
+    private InMemUserRepository userRepository;
+
+    @BeforeEach
+    void setUp(){
+        userRepository =  new InMemUserRepository();
```

Comment:
Is it necessary for each test to init this repository?

Reply:
removed